### PR TITLE
Add policy file to create a IAM user. Remove provisioned concurrency …

### PR DESCRIPTION
**Pull request by zhizhongk**
…for ecs lambda function

*Issue #, if available:*
Add policy file for customers who don't have admin access to their aws accounts. 

Remove provisioned concurrency config for ecs lambda function. Lambda version and alias will not be updated as the lambda function got updated. In this case, if customers make change on the function, then provisioned function won't be updated. 

For now the solution is to remove the provisioned concurrency config. Since the function memory is set to 128mb by default, cold start is not concerning compared to 3gb function.

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -111,11 +111,37 @@ wget https://github.com/aws-samples/amazon-chime-voiceconnector-transcription/re
...
+```
+aws iam create-user --user-name chime-transcription-deployment-engineer
+```
+
+Create deployment policy.
+```
+aws iam create-policy --policy-name chime-transcription-deployment-policy --policy-document file://permission/policy.json
+```
+
+Attach the policy to the user.
```
Comment (sachgo) [inline README.md:128]: May be call it work with AWS account admin and get the necessary permissions, account admin can use this command or can manually grant the permission using the attached policy file.

File: README.md [hunk 2]
```diff
@@ -111,11 +111,37 @@ wget https://github.com/aws-samples/amazon-chime-voiceconnector-transcription/re
...
+aws iam create-access-key --user-name chime-transcription-deployment-engineer
+```
+
+You have finished creating a user with deployment permissions.
 ### Deploying Sample
 
 Use [AWS Command Line Interface](https://aws.amazon.com/cli/) to deploy the sample
 
-Configure AWS Command Line Interface
+Configure AWS Command Line Interface. You can configure your access key from last step.
```
Comment (sachgo) [inline README.md:144]: you can use access key from last step to configure AWS CLI

File: README.md (hunk 1 above)
Comment (zhizhongk) [reply README.md:128]: Done

Comment (zhizhongk) [reply README.md:128]: done

File: README.md (hunk 2 above)
Comment (zhizhongk) [reply README.md]: done

**Comment by zhizhongk** [comment]
Addressed your comments

File: README.md [hunk 3]
```diff
@@ -111,11 +111,36 @@ wget https://github.com/aws-samples/amazon-chime-voiceconnector-transcription/re
 >>/usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
 >>```
 
+### (Optional) Configure AWS IAM User
+
+If administrative credential is not available, AWS account admin can use the following command to create a user for sample deployment, or grant the permission using the attached policy file.
```
Comment (sachgo) [inline README.md:116]: sorry may be put the link to the policy file here.

Comment (zhizhongk) [reply README.md]: Done in the latest pr

**Pull request merged**